ddtrace/internal/module.py

```diff
@@ -412,15 +413,14 @@ def find_spec(
 
     @classmethod
     def _check_installed(cls) -> None:
-        if not cls.is_installed():
-            raise RuntimeError("%s is not installed" % cls.__name__)
+        if cls.is_installed():
+            return
```

Comment:
Maybe? Or is this too noisy 😅 
```suggestion
        if cls.is_installed():
            log.debug("%s is not installed" % cls.__name__)
            return
```